doc/classes/Material.xml

```diff
@@ -5,6 +5,9 @@
 	</brief_description>
 	<description>
 		Material is a base [Resource] used for coloring and shading geometry. All materials inherit from it and almost all [VisualInstance3D] derived nodes carry a Material. A few flags and parameters are shared between all material types and are configured here.
+		[b]Note:[/b]  For performance reasons the [signal Resource.changed] signal, it's only emitted when these properties are changed:
+		- Name (inherited from [Resource])
+		- Shader
```

Comment:
```suggestion
		[b]Note:[/b] For performance reasons the [signal Resource.changed] signal is only emitted when the name and the shader of the material are changed.
```

Although these should ideally link to specific properties or methods.

Reply:
I can't find usages of something like properties linking, like `[property Resource.resource_name]` in the rest of the docs, does that works? If not, what would be the best approach to this linking?

Also, the material change is only done for ShaderMaterials, and does nothing in non-editor builds, so maybe this clarification in Material is actually pointless.

Reply:
It's `[member property_name]` not `[property property_name]`.

Reply:
Yes.